TestApp/Source/Shared/InputManager.h

```diff
@@ -2,6 +2,8 @@
 
 #include <Babylon/Runtime.h>
 #include <napi/napi.h>
+#include <napi/env.h>
```

Comment:
what's these for?

Reply:
env is needed in the header
https://github.com/BabylonJS/BabylonNative/blob/750b447dea40a4b6cf5936f462023e7b7fd5c3d7/TestApp/Source/Shared/InputManager.h#L64